launch/tier4_control_launch/launch/control.launch.xml

```diff
@@ -0,0 +1,251 @@
+      <composable_node pkg="predicted_path_checker" plugin="autoware::motion::control::predicted_path_checker::PredictedPathCheckerNode" name="predicted_path_checker">
+        <remap from="~/input/objects" to="/perception/object_recognition/objects"/>
+        <remap from="~/input/reference_trajectory" to="/planning/scenario_planning/trajectory"/>
+        <remap from="~/input/current_accel" to="/localization/acceleration"/>
+        <remap from="~/input/odometry" to="/localization/kinematic_state"/>
+        <remap from="~/input/predicted_trajectory" to="/control/trajectory_follower/lateral/predicted_trajectory"/>
+        <param from="$(var vehicle_param_file)"/>
+        <param from="$(var predicted_path_checker_param_path)"/>
+        <extra_arg name="use_intra_process_comms" value="$(var use_intra_process)"/>
+      </composable_node>
+    </load_composable_node>
+
+    <!-- control evaluator -->
+    <load_composable_node target="/control/control_container">
+      <composable_node pkg="autoware_control_evaluator" plugin="control_diagnostics::controlEvaluatorNode" name="control_evaluator">
+        <remap from="~/input/diagnostics" to="/diagnostics"/>
+        <remap from="~/input/odometry" to="/localization/kinematic_state"/>
+        <remap from="~/input/acceleration" to="/localization/acceleration"/>
+        <remap from="~/input/trajectory" to="/planning/scenario_planning/trajectory"/>
+        <remap from="~/metrics" to="/diagnostic/control_evaluator/metrics"/>
+        <remap from="~/input/vector_map" to="/map/vector_map"/>
+        <remap from="~/input/route" to="/planning/mission_planning/route"/>
+      </composable_node>
+    </load_composable_node>
+  </group>
+
+  <group>
+    <push-ros-namespace namespace="control"/>
+
+    <!-- control validator checker -->
+    <node_container pkg="rclcpp_components" exec="$(var container_executable)" name="control_validator_container" namespace="">
+      <composable_node pkg="autoware_control_validator" plugin="autoware::control_validator::ControlValidator" name="control_validator">
+        <remap from="~/input/kinematics" to="/localization/kinematic_state"/>
+        <remap from="~/input/reference_trajectory" to="/planning/scenario_planning/trajectory"/>
+        <remap from="~/input/predicted_trajectory" to="/control/trajectory_follower/lateral/predicted_trajectory"/>
+        <remap from="~/output/validation_status" to="~/validation_status"/>
+        <param from="$(var control_validator_param_path)"/>
+      </composable_node>
+
+      <composable_node pkg="glog_component" plugin="GlogComponent" name="glog_validator_component"/>
+    </node_container>
+  </group>
+
+  <group if="$(eval &quot;'$(var trajectory_follower_mode)' == 'smart_mpc_trajectory_follower'&quot;)">
+    <node pkg="autoware_smart_mpc_trajectory_follower" exec="pympc_trajectory_follower.py" name="pympc_trajectory_follower"/>
+  </group>
+</launch>
```

Comment:
```suggestion
  <group if="$(eval &quot;'$(var trajectory_follower_mode)' == 'smart_mpc_trajectory_follower'&quot;)">
    <node pkg="autoware_smart_mpc_trajectory_follower" exec="pympc_trajectory_follower.py" name="controller_node_exe"/>
  </group>
</launch>
```

`controller_node_exe` is correct? please check

![image](https://github.com/autowarefoundation/autoware.universe/assets/39142679/01ad1731-59ef-40cf-b2dd-1b6804573286)